TaskSharper.Calender.WPF/ViewModels/MonthViewModels/CalendarDateDayViewModel.cs

```diff
@@ -115,5 +131,11 @@ public void UpdateDate(DateTime date)
                 _logger.Error(e, "Failed to update view");
             }
         }
+
+        public void GoToDayView()
```

Comment:
Command is named GoToTodayViewCommand and functionen is named GoToDayView. Maby the Today view should be renamed to DayView instead?